src/modules/control/src/waypoint_rviz_visualizer.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+"""
+NOTE: Generates two circles in rviz to represent the donut waypoint selector in the control stack.
+      ---------
+      Topic: /virtual_donut_visualization
+        Msg: visualization_msgs/Marker
+"""
+
+import rospy
+import math
+from visualization_msgs.msg import Marker
+from geometry_msgs.msg import Point
+
+class VirtualDonut(object):
+    """Publishes a donut in rviz from parameters in the control launch file"""
+
+    def __init__(self):
+        """Initializing MarkerArray message publisher"""
+        self.innerDonutPublisher = rospy.Publisher("/inner_donut_visualization", Marker, queue_size=1)
+        self.outerDonutPublisher = rospy.Publisher("/outer_donut_visualization", Marker, queue_size=1)
+        
+        self.point_initialization_state = True
+
+        self.marker_initialization()
+    
+    def vdonut_callback(self, event):
```

Comment:
Is this function definition necessary? What is the drawback of using `create_virtual_donut()` as the callback?

Reply:
I'll change it tomorrow, just realized I derped while writing the code